go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.5.0
 	github.com/Microsoft/hcsshim v0.8.17
+	github.com/containerd/containerd v1.5.1
```

Comment:
I'd rather not add a dependency on `containerd`. Great code but also a big code base. Was `RunningInUserNS` removed from runc?

Reply:
It think it's still available in 
```
	"github.com/opencontainers/runc/libcontainer/userns"
```

Reply:
Ok, maybe I missed it.

Reply:
I grepped through an older version of runc